code/datums/components/butchering.dm

```diff
@@ -27,13 +27,29 @@
 		if(butchering_enabled && (can_be_blunt || source.get_sharpness()))
 			INVOKE_ASYNC(src, .proc/startButcher, source, M, user)
 			return COMPONENT_ITEM_NO_ATTACK
+	if(user.a_intent == INTENT_HARM && ishuman(M) && source.get_sharpness())
+		var/mob/living/carbon/human/H = M
+		if((H.health <= H.crit_threshold || (user.pulling == H && user.grab_state >= GRAB_NECK) || H.IsSleeping()) && user.zone_selected == BODY_ZONE_HEAD)
+			INVOKE_ASYNC(src, .proc/startNeckSlice, source, H, user)
+			return COMPONENT_ITEM_NO_ATTACK
 
 /datum/component/butchering/proc/startButcher(obj/item/source, mob/living/M, mob/living/user)
 	to_chat(user, "<span class='notice'>You begin to butcher [M]...</span>")
 	playsound(M.loc, butcher_sound, 50, TRUE, -1)
 	if(do_mob(user, M, speed) && M.Adjacent(source))
 		Butcher(user, M)
 
+/datum/component/butchering/proc/startNeckSlice(obj/item/source, mob/living/carbon/human/H, mob/living/user)
+	H.visible_message("<span class='danger'>[user] is slitting [H]'s throat!</span>", \
+					"<span class='userdanger'>Your throat is being slit by [user]!</span>", \
+					"<span class='userdanger'>Something is cutting into your neck!</span>")
+	playsound(H.loc, butcher_sound, 50, TRUE, -1)
+	if(do_mob(user, H, CLAMP(500 / source.force, 30, 50)) && H.Adjacent(source))
+		H.visible_message("<span class='danger'>[user] slits [H]'s throat!</span>", \
+					"<span class='userdanger'>[user] slits your throat...</span>")
+		H.adjustBruteLoss(source.force, 0)
+		H.bleed_rate = CLAMP(H.bleed_rate + 20, 0, 30)
```

Comment:
A bleed rate of 20 will result in the victim loosing 390 units of blood, not accounting for natural blood regeneration, which, over 40 ticks, ammounts to 20 units of blood, assuming the victim is well fed. So we are down to 370 units of blood lost. Normal blood volume is 560, meaning the victim will have 190 units of blood left once he stops bleeding.

190 units of blood falls into BLOOD_VOLUME_BAD, which causes 5 units of oxyloss per tick. Respiration restores 5 units every 4 ticks. In addition the victim has a 15% chance per tick of falling unconscious, will therefore happen once every 7 ticks.

At 3.75 oxyloss per tick (this accounts natural oxyloss regeneration), it takes 54 ticks to die. This neglects the fact that the victim will be dying faster once he hits crit. It also neglets the brute damage done by this maneuver. During 54 ticks, the victim will generate 27 units of blood, bringing him up 217 units, which is still BLOOD_VOLUME_BAD.

We will hit BLOOD_VOLUME_BAD 25 ticks in. Neglecting the oxyloss that occurs before BLOOD_VOLUME_BAD, getting your throat slit will kill you in 79 ticks. Asuming 1.3 seconds per tick (The brain tumor quirk needs 1000 ticks to kill you, which I know from experience to be about 22 minutes), this will kill you in less than 1 minute and 40 seconds.

Of course, the bandages, an item available in the white medkit, will stop all bleeding. So, assuming the victim escapes immediately after his or her throat has been slit, they have about 30 seconds to find bandages. Best equipped to help the victim would be a mediborg, who has bandages, to stop the bleeding, saline-glucose solution, to reduce the severity of bloodloss, lollipops with iron and omnizine to both replace blood, and cure oxyloss damage.

Getting your throat slit is guranteed death, unless you have bandages on your person, or receive immediate medical aid. Another approach would be taking heparin, which caps your bleed rate at 8, which results in only 60 units of blood lost, assuming it is in your system for 1 tick only.